site/docs/master/namespace.md

```diff
@@ -1,23 +1,19 @@
-# Run in custom namespace
+# Run in a custom namespace
 
-You can run Velero in any namespace.
+The Velero installation and backups by default are run in the `velero` namespace. However, it is possible to customize it.
 
-First, ensure you've [downloaded & extracted the latest release][0].
+### 1) Customize the namespace on the cluster 
 
-Then, install Velero using the `--namespace` flag:
-
-```bash
-velero install --bucket <YOUR_BUCKET> --provider <YOUR_PROVIDER> --namespace <YOUR_NAMESPACE>
-```
+Use the `--namespace` flag, in conjunction with the other flags in the `velero install` command (as shown in the [the Velero install instructions][0]). This will inform Velero where to run.
 
+### 2) Customize the namespace for operational commands
 
-
-## Specify the namespace in client commands
-
-To specify the namespace for all Velero client commands, run:
+To have namespace consistency, specify the namespace for all Velero operational commands same as the namespace used to install Velero:
```

Comment:
perhaps "...for all Velero operational commands to be the same as the namespace..."